CHANGELOG.md

```diff
@@ -16,6 +16,25 @@ If you run into any issues with the upgrade or need to downgrade to a previous v
 2. Modify the `bootstrappers` config in the M3DB YAML file from `filesystem, commitlog, peers, uninitialized_topology` to `filesystem, peers, commitlog, uninitialized_topology`. This will force the node to bootstrap from its peers instead of the local snapshot and commitlog files it has on disk, bypassing any issues related to file incompatibility between versions.
 3. Turn the node back on and wait for it to finish bootstrapping and snapshotting. Once everything looks stable, change the config back to `filesystem, commitlog, peers, uninitialized_topology` so  that the next time the node is restarted it will default to using the snapshot and commitlog files.
 
+## New Features
+
+- **M3DB**: Restructuring of commitlog and snapshotting feature as described above (#1384)
```

Comment:
Would be good to have it in the docs, then link here rather than saying `described above`

Reply:
same comment as before, should be transparent upgrade for everyone anyways